### PR TITLE
Update dependencies & allow admins to edit actions. 

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
This pull request updates dependencies (notably, Webpack 4 for less buggy re-builds) and adds the ability for admins to edit in-progress actions (which we'll want for [#165160940](https://www.pivotaltracker.com/story/show/165160940)).

As part of that, I updated this app's `window.AUTH` to be an object (filled in by a [view composer](https://laravel.com/docs/5.6/views#view-composers)) so that we'd have access to the user's role. This follows the same approach we've used in Phoenix.

#### How should this be reviewed?
I think everything's good, but can't hurt to click around on QA a bit before we deploy this!

#### Any background context you want to provide?
nah.

#### Relevant tickets
[#165160940](https://www.pivotaltracker.com/story/show/165160940)

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: resources/assets/components/Action/index.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ class Action extends React.Component {
   render() {
     const action = this.props.action;
     const campaign = this.props.campaign;
+    const campaignInProgress = parse(campaign.start_date) > Date.now();
```
Comment (katiecrane) [inline resources/assets/components/Action/index.js]: I've been confused about dates recently, but wouldn't a campaign be "in progress" if it's start date is `<` now? (meaning it has already passed?)

Comment (DFurnes) [reply resources/assets/components/Action/index.js]: Oh yeah this is very wrong. 🙃

Comment (ngjo) [reply resources/assets/components/Action/index.js]: This probably doesn't have anything to do with what it's like in Rogue, but I know that Gambit relies on this date and I think there is something funny with how they "read" the date too and treat it as closed?

Comment (DFurnes) [reply resources/assets/components/Action/index.js]: Updated this in 47b1a2d! Thanks for catching @katiecrane!

Comment (DFurnes) [reply resources/assets/components/Action/index.js]: Maybe we should just add a computed "open" boolean on the campaign response that does this for us so we don't have to risk silly mistakes like this in multiple places. 😂

Comment (DFurnes) [reply resources/assets/components/Action/index.js]: I tested this a little more thoroughly this time and can confirm that it works as expected now: admins can edit actions for open & closed campaigns, but staffers only see the "edit" and "delete" buttons for campaigns that haven't started yet.

Comment (katiecrane) [reply resources/assets/components/Action/index.js]: Oooo I love the computed boolean idea!!

**Pull request merged**